### PR TITLE
Change logo on projects page from MLH Fellowship to Prep

### PR DESCRIPTION
**Pull request by chiuannica**
## Please, go through these steps before you submit a PR.

- [X] My Pod Leader knows I'm working on this Pull Request
- [X] I've explained what the Pull Request is adding.
- [X] I've explained why this is important.

This pull request is changing the MLH Fellowship logo (svg) to the MLH Prep logo (png). This is important because the prep logo is used on the home page, and there should be consistency throughout the site.

I changed it by changing logo.svg to mlh-prep.png. Since they are different formats, the new logo needed to be resized to fit properly. I used the class "w-100" from Bootstrap to make the width 100% of the area and fit in above "Projects" properly.

Wide screen
Screen Shot 2022-03-08 at 10 09 12 PM

Mid-size screen
Screen Shot 2022-03-08 at 9 54 03 PM

Narrow screen
Screen Shot 2022-03-08 at 10 08 12 PM

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <div class="container">
         <div class="logo">
             <a href="/">
-                <img src="/assets/img/logo.svg"/>
+                <img class="w-100" src="/assets/img/mlh-prep.png"/>
```
Comment (sohamsshah) [inline _layouts/page.html]: Having `alt` attribute in the `img` tags would be great in terms of accessibility. Also, to have better [CLS](https://web.dev/cls/), it is a good practice to add `width` and `height` attributes to the images in the DOM.

**Review by sohamsshah — CHANGES_REQUESTED** [review]
Hey @chiuannica , great work 🥳.
Since you are working for the logo image, I have suggested a small enhancement. Would like to know your views on that!

File: _layouts/page.html (hunk 1 above)
Comment (marcnjaramillo) [inline _layouts/page.html]: @sohamsshah makes a good point about accessibility. Here is a good resource for [alt text](https://webaim.org/techniques/alttext/) and a well thought-out post on [StackOverflow](https://stackoverflow.com/questions/23526791/when-is-it-acceptable-to-leave-the-alt-text-empty-on-an-html-image-if-ever) if you would like to learn more about it. Note that the alt text doesn't need to be overly descriptive; something like "mlh prep program logo" would be sufficient.

**Review by marcnjaramillo — CHANGES_REQUESTED** [review]
This is an excellent start. Thank you for paying close attention to the appearance of the logo for different screen widths. I left one comment, and once you address that we'll be ready to merge.

File: _layouts/page.html (hunk 1 above)
Comment (chiuannica) [reply _layouts/page.html]: Thanks for the input, I overlooked that!

Comment (sohamsshah) [reply _layouts/page.html]: Awesome 🙌👍

**Review by marcnjaramillo — APPROVED** [review]
Great work!

**Pull request merged**